src/lib/helpers.js

```diff
@@ -0,0 +1,24 @@
+function isCommandToken(obj) {
```

Comment:
I've got a slightly different approach to this, but let's adopt this one for now and consider the other style separately.